usr.sbin/bsdinstall/scripts/hostname

```diff
@@ -40,6 +40,9 @@ f_include $BSDCFG_SHARE/dialog.subr
 # Default value
 #
 : ${HOSTNAME=$( hostname )}
+if [ "X$HOSTNAME" = "X" ]; then
+	HOSTNAME="morello"
+fi
```

Comment:
cheribuild sets `hostname` in `rc.conf` to a cheribuild OS target, e.g. `cheribsd-morello-purecap`. Should we make it consistent?

We should make `HOSTNAME` depend on `uname -p`.

Reply:
Or be lazy and default it to cheribsd. Don't want to use the cheribuild architecture names though, in a world of installers and packages those cheribuildisms should never appear.

Reply:
Went with ‘cheribsd’ (but not ‘cherubs’ as proposed by macOS).